lib/src/stores/wallet/wallet_store.dart

```diff
@@ -131,7 +131,7 @@ abstract class WalletStoreBase with Store {
     if (value.isEmpty) {
       isValid = true;
     } else {
-      const pattern = '^([0-9]+([.][0-9]{0,9})?|[.][0-9]{1,9})\$';
+      const pattern = '^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\$|^[-0-9a-zA-Z.]{1,253}\$';
```

Comment:
This regex is used to match an Oxen amount, not an IP

Reply:
Good grief. How embarrassing.

Reply:
OK, this is fixed now. Sorry.

For good measure, I also fixed the simplistic regex for determining a legal port number, although regex really isn't the right way to validate an integer.